metaci/build/management/commands/tests/test_run_build.py

```diff
@@ -0,0 +1,42 @@
+import os
+from unittest import mock
+
+import pytest
+from django.test import TestCase
+
+from metaci.build.management.commands.run_build import Command
+from metaci.conftest import (
+    BranchFactory,
+    OrgFactory,
+    PlanFactory,
+    PlanRepositoryFactory,
+    RepositoryFactory,
+    UserFactory,
+)
+
+
+@mock.patch("metaci.build.management.commands.run_build.run_build")
+@pytest.mark.django_db
+class TestRunBuild(TestCase):
+    # def test_run_build_scratch(self, run_build):
+    #     self._testrun_build(run_build, True)
+
+    def test_run_build_persistent(self, run_build):
+        self._testrun_build(run_build, False)
+
+    def test_run_build_persistent_2(self, run_build):
+        self._testrun_build(run_build, False)
+
+    def test_run_build_persistent_3(self, run_build):
+        self._testrun_build(run_build, False)
```

Comment:
What's the purpose of doing the same test 3 times?

Reply:
I was having a weird crashing behaviour when I ran more than one test. I got so fixated on fixing the crash that I forgot to put the tests back to what they were originally testing. In the end it was because I was trying to hook into the Command API at a low-level which would run through all of the argument validation code etc. but that low-level was also resetting Django state. It was a while between when I wrote the call to the low-level API and when it started causing trouble (because I originally had only a single unit test) so it wasn't immediately obvious to me that that was the source of the problem. Anyhow, I got distracted fixing that and forgot to put my tests back to their original state.